cdk/README.md

```diff
@@ -0,0 +1,49 @@
+Pre-requisites
+1) Create secret in Secrets Manager as per https://docs.aws.amazon.com/AmazonECS/latest/developerguide/linux-gmsa.html#linux-gmsa-setup
+2) 'default' AWS profile
+
+Steps to run tasks in ECS with Credentials-fetcher.
```

Comment:
Please mention whether this is domainless gMSA or domain-joined gMSA

Reply:
Will do